content/docs/hooks/use-clipboard.mdx

```diff
@@ -37,7 +38,7 @@ import { useClipboard } from '@chakra-ui/react'
 ```jsx
 function Example() {
   const [value, setValue] = React.useState('Hello world')
```

Comment:
Let's remove this line and obtain the `value` variable from the `useClipboard` below.